pages/osx/afplay.md

```diff
@@ -0,0 +1,19 @@
+# afplay
+
+> Command-line audio player.
+
+- Play a sound file (waits until playback ends):
+
+`afplay {{path/to/file}}`
+
+- Play a sound file at 2x speed:
```

Comment:
Maybe mention "reproduction speed" or "rate", or some other word starting with "r", to help users develop mnemonics to learn the command.

Reply:
Interesting idea, but all of the alternatives I can think of sound worse to me. I don't think I've ever seen "reproduction" used to mean audio playback. "rate" makes sense semantically but I think "speed" will be easier to understand for a novice user.

Reply:
Sure. You're correct that it's not always possible/desirable. One strategy we use often is parenthesis — so here it could be e.g. `at 2x speed (playback rate)`, or `at 2x the normal rate (speed)`, or some other variant. WDYT?

Reply:
"playback rate" is almost there, though even that still sounds a bit weird to me. We could also use the longform `--rate` option, like you suggested below for `--time`.

Reply:
Given that the option itself is `rate`, not `speed`, I am fine with using it.

Reply:
For future reference, "playback rate" and the full option name were added in [`cd068a4`](https://github.com/tldr-pages/tldr/pull/3403/commits/cd068a4b2cc341d3d602050e40dbec482956d4d0).